x-pack/plugins/cases/common/types/domain/connector/mappings.v1.test.ts

```diff
@@ -5,9 +5,7 @@
  * 2.0.
  */
 
-import { ConnectorMappingsAttributesPartialRt } from '../../../server/common/types/connector_mappings';
-import { decodeOrThrow } from '../runtime_types';
-import { ConnectorMappingsAttributesRt, ConnectorMappingsRt } from './mappings';
+import { ConnectorMappingsAttributesRt, ConnectorMappingsRt } from './v1';
```

Comment:
in the mapping tests we import these from `'./v1'` instead of `from './mappings.v1'` so another thing that makes me think this file is redundant and just introduces a bit of confusion with the imports 😄

Reply:
Ok, I will merge them into the same file.